moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -265,17 +265,26 @@ export const PROJECT_LIST_VIEW_FILTERS_CONFIG = {
       lookup: {
         table_name: "moped_fund_sources",
         getOptionLabel: (option) => option.funding_source_name,
-        operators: [
-          "string_equals_case_insensitive",
-          "string_does_not_equal_case_insensitive",
-        ],
+        operators: ["string_contains_case_insensitive"],
+      },
+      operators: [
+        "string_contains_case_insensitive",
+        "string_is_null",
+        "string_is_not_null",
+      ],
+    },
+    {
+      name: "funding_source_and_program_names",
```

Comment:
thanks for hitting the brakes when you realized that we can't have two identical names in the config! I reproduced the quirk that you mentioned locally, and I see how the code does not like having two identical names and starts populating source options for programs. 🫠

How do you feel about keeping the changes to the funding sources in this PR and then we regroup on the funding program advanced search? I'm thinking that we should add another column to the view that is a concatenation of funding program names like we have for funding sources for that work.

That way, we could advanced search for:
- sources
- programs
- sources and programs
- sources or programs
- projects with no sources (Funding sources is null)
- projects with no programs (Funding programs is null)

I could use another set of eyes to see if this sounds right!

Reply:
Here is the spinoff issue https://github.com/cityofaustin/atd-data-tech/issues/18763